code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -70,8 +70,8 @@
 		/obj/item/reagent_containers/food/snacks/grown/watermelon = list("watermelonjuice" = 0),
 		/obj/item/reagent_containers/food/snacks/watermelonslice = list("watermelonjuice" = 0),
 		/obj/item/reagent_containers/food/snacks/grown/berries/poison = list("poisonberryjuice" = 0),
+		/obj/item/reagent_containers/food/snacks/grown/pumpkin/blumpkin = list("blumpkinjuice" = 0), //order is important here as blumpkin is a subtype of pumpkin, if switched blumpkins will produce pumpkin juice
 		/obj/item/reagent_containers/food/snacks/grown/pumpkin = list("pumpkinjuice" = 0),
-		/obj/item/reagent_containers/food/snacks/grown/pumpkin/blumpkin = list("blumpkinjuice" = 0),
 		/obj/item/reagent_containers/food/snacks/grown/apple = list("applejuice" = 0),
 		/obj/item/reagent_containers/food/snacks/grown/grapes = list("grapejuice" = 0),
 		/obj/item/reagent_containers/food/snacks/grown/grapes/green = list("grapejuice" = 0),
```

Comment:
Don't think like 77 here is necessary anymore

Reply:
It is not. Fixed on commit [5c845e3](https://github.com/ParadiseSS13/Paradise/pull/21947/commits/5c845e3253b0cbe74a749542f083cfa3c77b9982)